relay/kusama/src/lib.rs

```diff
@@ -1743,6 +1794,7 @@ pub mod migrations {
 		>,
 		pallet_im_online::migration::v1::Migration<Runtime>,
 		parachains_configuration::migration::v7::MigrateToV7<Runtime>,
+		runtime_parachains::scheduler::migration::v1::MigrateToV1<Runtime>,
```

Comment:
This is some blinking migration, which I'm not sure about - it has been copypasted, then commented, then removed. But according to https://github.com/polkadot-fellows/runtimes/pull/56#pullrequestreview-1715355365, it still need to be added iiuc

Reply:
This migration was already executed on Kusama and Polkadot as well when we finally have the update applied there.

Reply:
ok, removed